### PR TITLE
Odim 5930: Unit test coverage for agcommon under svc-aggregation

### PR DESCRIPTION
**Pull request by lpooja**
Updated Test case for svc-aggregation

### CODE REVIEW

File: svc-aggregation/agcommon/common_test.go [hunk 1]
```diff
@@ -1,27 +1,43 @@
 //(C) Copyright [2020] Hewlett Packard Enterprise Development LP
+
 //
+
```
Comment (jeevan-kamkar) [inline svc-aggregation/agcommon/common_test.go]: remove the newly added lines

File: svc-aggregation/agcommon/common_test.go [hunk 2]
```diff
@@ -85,3 +117,137 @@ func stubDeleteConnectionMethod(table, key string, dbtype common.DbType) *errors
...
+	// assert.Equal(t, "lookup 10.0.0: no such host", err.Error(), "If invalid ip, no such host should be given")
+	LookupIPfunc = func(host string) (ip []net.IP, err error) {
+		err = fmt.Errorf("error")
+		return
+	}
+	ip, _, _, err := LookupHost("10.0.0")
+	assert.NotNil(t, err, "There should be an error")
+
+	LookupIPfunc = func(host string) (ip []net.IP, err error) {
+		// ip=append(ip, net.ParseIP("192.100.10.4"))
```
Comment (jeevan-kamkar) [inline svc-aggregation/agcommon/common_test.go]: remove the commented code

**Review by jeevan-kamkar — CHANGES_REQUESTED** [review]
address the requested changes

File: svc-aggregation/agcommon/common_test.go (hunk 1 above)
Comment (lpooja) [reply svc-aggregation/agcommon/common_test.go]: removed

File: svc-aggregation/agcommon/common_test.go (hunk 2 above)
Comment (lpooja) [reply svc-aggregation/agcommon/common_test.go]: removed

File: svc-aggregation/agcommon/common_test.go [hunk 3]
```diff
@@ -85,3 +105,129 @@ func stubDeleteConnectionMethod(table, key string, dbtype common.DbType) *errors
...
+	type args struct {
+		plugin string
+	}
+	tests := []struct {
+		name  string
+		args  args
+		want  int
+		want1 bool
+	}{
+		// TODO: Add test cases.
```
Comment (chauberahul1993) [inline svc-aggregation/agcommon/common_test.go]: No test case Added, Remove if not required

**Review by hemkapoor — APPROVED** [review]

File: svc-aggregation/agcommon/common_test.go (hunk 3 above)
Comment (lpooja) [reply svc-aggregation/agcommon/common_test.go]: removed

**Review by chauberahul1993 — APPROVED** [review]

**Review by jeevan-kamkar — APPROVED** [review]

**Review by amar-shalgar — APPROVED** [review]

**Pull request merged**